src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/WebJira.kt

```diff
@@ -107,6 +107,7 @@ data class WebJira(
     }
 
     internal fun administrate(): JiraAdministrationMenu {
+        NotificationPopUps(driver).waitUntilAuiFlagsAreGone()
```

Comment:
Now I'm looking at this change again I think it can be problematic in the current form.

If I understand the code correctly it first waits up to 10 seconds for the flags to close themselves and only then it clicks them.

It means that when someone will want to use thi in an measured action it will have unpretictable results.

Currently it's only used in `SET_UP` measure which we don't judge upon, but it's fairly easy to use this menu in other measures without realising the problem. Should we safeguard future contributions in some way?

Reply:
Yeah, we should not wait for popups to appear, but close only the ones, which are already present.